src/pages/index.js

```diff
@@ -0,0 +1,365 @@
+import "./index.css";
+import avatar from "../images/avatar.jpg";
+import plusIcon from "../images/plus.svg";
+import logoSrc from "../images/logo.svg";
+import cross from "../images/white-cross.svg";
+import closeCross from "../images/cross.svg";
+import pencilIcon from "../images/pencil.svg";
+import whitePencilIcon from "../images/white-pencil.svg";
+import heartLikeIcon from "../images/heart-like.svg";
+import heartLikeIconHover from "../images/heart-like-hover.svg";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+} from "../scripts/validation.js";
+import Api from "../utils/Api.js";
+
+const avatarImage = document.getElementById("avatarIcon");
+if (avatarImage) {
+  avatarImage.src = avatar;
+}
+
+const pencilImage = document.getElementById("pencil");
+if (pencilImage) {
+  pencilImage.src = pencilIcon;
+}
+
+const logoImage = document.getElementById("logo");
+if (logoImage) {
+  logoImage.src = logoSrc;
+}
+
+const plusImage = document.getElementById("plus-icon");
+if (plusImage) {
+  plusImage.src = plusIcon;
+}
+
+const crossOne = document.getElementById("cross");
+if (crossOne) {
+  crossOne.src = cross;
+}
+
+const crossTwo = document.getElementById("closeCross");
+if (crossTwo) {
+  crossTwo.src = closeCross;
+}
+
+const avatarEditIcon = document.querySelector(".profile__avatar-edit-icon");
+if (avatarEditIcon) {
+  avatarEditIcon.src = whitePencilIcon;
+}
+
+// DOM elements for profile
+const profileName = document.querySelector(".profile__name");
+const profileDescription = document.querySelector(".profile__description");
+const avatarEditButton = document.querySelector(".profile__avatar-edit");
+const avatarEditModal = document.querySelector("#avatar-edit-modal");
+const avatarForm = document.querySelector("#avatar-form");
+const avatarInput = document.querySelector("#avatar-input");
+const avatarModalCloseBtn = avatarEditModal.querySelector(
+  ".modal__close-button"
+);
+
+// API initialization
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "528aeb29-13e8-4b53-8154-7e2f9458e435",
+    "Content-Type": "application/json",
+  },
+});
+
+avatarForm.addEventListener("submit", (evt) => {
+  evt.preventDefault();
+
+  const avatarUrl = avatarInput.value;
+
+  const submitButton = avatarForm.querySelector(".modal__submit-button");
```

Comment:
Please, don't search the submit button every time you submit a form. It's a waste of resources

You have use `evt.submitter`  as the **submit button** inside every submit handler, to avoid searching it

```js

  const submitButton = evt.submitter
```

**Please, correct it everywhere in the code**